examples/js/postprocessing/EffectComposer.js

```diff
@@ -203,3 +203,45 @@ Object.assign( THREE.Pass.prototype, {
 	}
 
 } );
+
+// Helper for passes that need to fill the viewport with a single quad.
+THREE.Pass.FullScreenQuad = (function() {
+
+	var _camera = new THREE.OrthographicCamera( - 1, 1, 1, - 1, 0, 1 );
+	var _geometry = new THREE.PlaneBufferGeometry( 2, 2 );
```

Comment:
I don"t think the underscores are needed now.

Reply:
Amended the last commit to remove the underscores from the variables in the closure.